### PR TITLE
[arch/drivers] renames

### PR DESCRIPTION
**Pull request by amirgon**
See: https://github.com/lvgl/lvgl/issues/2720#issuecomment-1321242462

### CODE REVIEW

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -68,7 +68,7 @@
 - **ARM2D GPU support** Get support for Arm's Microcontroller 2D Graphics Acceleration, e.g. Helium based acceleration, DMA-350 based acceleration etc.
 - **PubSub messaging** A publisher-subscriber based messaging system is added to make communication between components easier. See more [here](https://docs.lvgl.io/master/others/msg.html).
 - **Pinyin IME** Add support for Pinyin IME Chinese input. See more [here](https://docs.lvgl.io/master/others/ime_pinyin.html).
-- **render_start_cb** A new callback is added to `lv_disp_drv_t` to indicate when the rendering starts. It's useful to make synchronization, e.g. wait for a TE signal.
```
Comment (kisvegabor) [inline docs/CHANGELOG.md]: This file should be reverted.

**Review by kisvegabor — COMMENTED** [review]
Thank you, only one note.

File: docs/CHANGELOG.md (hunk 1 above)
Comment (amirgon) [reply docs/CHANGELOG.md]: Done.

**Comment by kisvegabor** [comment]
Thank you!

**Pull request merged**